common/djangoapps/mailchimp_pipeline/tests/helpers.py

```diff
@@ -22,11 +24,12 @@ def create_organization(user):
     organization.save()
     return organization
 
+
 def create_organization_partner_object(user):
     """
-        create and return organziation partner with given user as an organization admin
-        :param user: organization admin user
-        :return: created organziation partner
+    create and return organization partner with given user as an organization admin
+    :param user: organization admin user
+    :return: created organization partner
```

Comment:
Lets use the same format for docstring here as well as mentioned above

Reply:
These aren't added by me, I only fixed the spell mistake.